solarized-definitions.el

```diff
@@ -1,5 +1,9 @@
 (eval-when-compile
-  (require 'cl))
+  (require 'cl-lib)
```

Comment:
Ah, it looks like a typo, since your earlier commit with the bad indentation had the right code.